sale_manual_delivery/wizard/manual_proc_view.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id='manual_delivery_wizard_form' model='ir.ui.view'>
+        <field name="name">manual.delivery.wizard.form</field>
+        <field name="model">manual.delivery</field>
+        <field name='arch' type='xml'>
+            <form string="Create Manually Delivery" name="manual_procurment_form">
+                <sheet>
+                    <group name="date">
+                        <field name='date_planned' required="1"/>
+                    </group>
+                    <group name="carrier">
```

Comment:
From the original PR: missing `partner_id` . This is mandatory for us in order to be able to change the place where it's going to be shipped.